apps/styleguide/src/forms/InputOntology.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <FormGroup v-bind="$props" v-on="$listeners">
+    <div class="p-0 m-0">
+      <InputOntologySubtree
+        class="pl-0"
+        :terms="terms"
+        :selection="selection"
+        :expanded="expanded"
+        @toggleExpand="toggleExpand"
+        @select="select"
+        @deselect="deselect"
+      />
+    </div>
+  </FormGroup>
+</template>
```

Comment:
combine with existing Tree component , code duplication

Reply:
Agreed. Did you assess if they can be drop in replacement of each other?

Reply:
> Agreed. Did you assess if they can be drop in replacement of each other?

If I look I get impression that difference is that this new component also takes care of the loading of data. This can be a nice benefit but should check if we also need a 'dumb' component that you can use without preloading and where you inject the data as a prop.

Reply:
merged